packages/vscode-extension/src/service/MessageService.ts

```diff
@@ -7,31 +7,47 @@ import { window } from 'vscode';
 import { Output } from '../suitecloud';
 import { COMMAND, SEE_DETAILS } from './TranslationKeys';
 import { VSTranslationService } from './VSTranslationService';
+import { EOL } from 'os';
```

Comment:
Looks like this isn't being used. Can we remove it?

Reply:
Sure. Removed.